### ISSUE TITLE
Format ville

### ISSUE THREAD

**Issue by Drakone** [pixelhumain/co2#831]
C'est pas très homogène.

![capture du 2018-04-01 18-18-05](https://user-images.githubusercontent.com/2474636/38174163-4db0ff3a-35da-11e8-8882-7a9a07987c1a.png)

**Comment by RaphaelRIVIERE** [comment]
Yep , faut faire une passe sur toute les villes est les mettres en minuscule
